Update docs to the style guide. (#35)
iotaledger/inx-dashboard: documentation/docs/welcome.md

# Welcome to INX-Dashboard

> INX-Dashboard provides a web GUI tool to help you manage your node. It provides an overview of the node's status and health and allows you to choose its peers, as well as the enabled plugins. It has a built-in explorer of all nodes on the network and visualizer for activity on the Tangle.

Reviewer: ```suggestion
The INX-Dashboard provides a web GUI tool to help you manage your node. It provides an overview of the node's status and health and allows you to choose its peers and the enabled plugins. It has a built-in explorer of all nodes on the network and a visualizer for activity on the Tangle.
```

Author: Since INX-Dashboard is a name, I don't think that "the" belongs there. I would rather keep "as well as" instead of "and" to indicate that peers and plugins are not related in any way.

Reviewer: It's a dashboard plugin, so it's either the dashboard or the plugin.

## Setup

We recommend you to use the [Docker images](https://hub.docker.com/r/iotaledger/inx-dashboard).

> These images are also used in the [Docker setup](http://wiki.iota.org/hornet/develop/how_tos/using_docker) of Hornet.

Reviewer: ```suggestion
The Hornet smart contract node also uses these images for the [Docker setup](http://wiki.iota.org/hornet/develop/how_tos/using_docker).
```

Author: I think here it works better in passive voice.
